portal/components/config.js

```diff
@@ -1,22 +1,32 @@
 const resourceConfig = {
   personas: [
-    {
-      consonant: true,
-      en: "school head",
-      hi: "स्कूल प्रमुख",
-      credentials: "e-Samwad",
-      applicationId: process.env.NEXT_PUBLIC_FUSIONAUTH_SCHOOL_APP_ID,
-      redirectUrl: "school",
-    },
+    // {
+    //   consonant: true,
+    //   en: "school head",
+    //   hi: "स्कूल प्रमुख",
+    //   credentials: "e-Samwad",
+    //   applicationId: process.env.NEXT_PUBLIC_FUSIONAUTH_SCHOOL_APP_ID,
+    //   redirectUrl: "school",
+    // },
     {
       consonant: false,
-      en: "official",
-      hi: "अधिकारी",
-      credentials: "Shiksha Saathi",
+      en: "Login",
+      hi: "लॉग इन",
+      suben: "official",
+      subhi: "अधिकारी",
+      credentials: "Admin Console",
       applicationId: process.env.NEXT_PUBLIC_FUSIONAUTH_STATE_APP_ID,
-      redirectUrl: `admin#/device_donation_donor`,
+      redirectUrl: `admin#/userSamiksha`,
     },
   ],
+  loginPerson: {
```

Comment:
Why the need to create a separate `loginPerson` object? Can we just use the existing `personas` array and add an `enabled` flag based on which to display it on the screen?